icons/domo.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Domo</title><path d="M0 0v8.787h2.124a3.203 3.203 0 0 1 3.142 2.567 3.206 3.206 0 0 1 6.282 0V8.787l3.205 3.205 3.214-3.205v2.567A3.206 3.206 0 0 1 24 10.61V0Zm8.407 9.854a2.138 2.138 0 1 0-.001 4.276 2.138 2.138 0 0 0 0-4.276zm12.7 0a2.138 2.138 0 1 0 0 4.275 2.138 2.138 0 0 0 0-4.275zm-20.045.004v4.267h1.202a2.138 2.138 0 0 0 0-4.267zm11.548 1.506v3.832h-1.062V12.63a3.205 3.205 0 0 1-6.282 0 3.19 3.19 0 0 1-.75 1.495 3.193 3.193 0 0 1-2.392 1.071H0V24h24V13.374a3.205 3.205 0 0 1-6.033-.744v2.566h-1.08v-3.832l-2.134 2.143z"/></svg>
```

Comment:
I'm seeing a difference with the right `O` comparing with their website header logo:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/36cb12f1-3a53-4647-9524-be49ee052d67)

Can you take another look?